frontend/src/components/store/PluginCard.tsx

```diff
@@ -113,8 +113,19 @@ const PluginCard: FC<PluginCardProps> = ({ plugin }) => {
             }}
             className="deckyStoreCardInfo"
           >
-            <p className={joinClassNames(staticClasses.PanelSectionRow)}>
-              <span>Author: {plugin.author}</span>
+            <p className={joinClassNames(staticClasses.PanelSectionRow)} style={{ marginTop: '0px' marginLeft: '16px'}}>
```

Comment:
```suggestion
            <p className={joinClassNames(staticClasses.PanelSectionRow)} style={{ marginTop: '0px', marginLeft: '16px'}}>
```
sorry, forgot a comma
next time, i think i'll test it properly before suggesting a change...

Reply:
> 

😂